<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A Lab Management System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: #ffffff;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0,0">
      <item row="3" column="1">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QWidget" name="time_table_layout" native="true">
          <property name="styleSheet">
           <string notr="true">*{
	background-color: #0388fc;
	border: none;
	border-radius: 20px;
}

QTextBrowser {
	background-color: #ffffff;
	color:	#000000;
	border:none;
	border-radius:10px;
}

/* Scroll bar */
QScrollBar
{
	width:10px;
	border: none;
    background: #0388fc;
}
QScrollBar::add-page, QScrollBar::sub-page 
{
	background: #0388fc;
}
QScrollBar::add-line, QScrollBar::sub-line 
{
	background: #0388fc;
}
QScrollBar::handle
{
	background-color: #fc7303;
	min-height: 30px;
	border-radius: 5px;
	border:none;
}
QScrollBar::up-arrow
{
	background: none;
}
QScrollBar::down-arrow
{
	background: none;
}</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item row="1" column="0" colspan="2">
            <widget class="QScrollArea" name="scrollArea">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>926</width>
                <height>460</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <layout class="QGridLayout" name="time_table"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QFrame" name="horizontalFrame">
        <property name="styleSheet">
         <string notr="true">*{
	background-color: #0388fc;
}

/* Button styling */
QPushButton {
	color: #ffffff;
	background-color: #0388fc;
	border-radius: 20px;
}
QPushButton::hover {
	border: 5px solid;
	border-color: #fc7303;
}
QPushButton::pressed {
	background-color: #fc7303;
	color: #ffffff;
}

/* Combo Box Styling */
QComboBox {
	color: #ffffff;
    background-color: #0388fc;
    border: 5px solid #0388fc;
	border-radius: 20px;
    padding-left: 20px;
}
QComboBox:hover {
    border: 5px solid #fc7303;
}
QComboBox:editable {
    background: #0388fc;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    border: none;
}
QComboBox QAbstractItemView {
	color: #ffffff;
	background-color: #0388fc;
    border: 5px solid #fc7303;
    padding: 10px;
}

/* Scroll bar */
QScrollBar
{
	width:10px;
	border: none;
    background: #0388fc;
}
QScrollBar::add-page, QScrollBar::sub-page 
{
	background: #0388fc;
}
QScrollBar::add-line, QScrollBar::sub-line 
{
	background: #0388fc;
}
QScrollBar::handle
{
	background-color: #fc7303;
	min-height: 30px;
	border-radius: 5px;
	border:none;
}
QScrollBar::up-arrow
{
	background: none;
}
QScrollBar::down-arrow
{
	background: none;
}</string>
        </property>
        <layout class="QHBoxLayout" name="top_bar">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="log_experiment">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Log Experiment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="manage_stock">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Manage Stock</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="date_range">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="account">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Account</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="help">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Help</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
